<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowContents</class>
 <widget class="QWidget" name="MainWindowContents">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string></string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="workLayout">
     <item>
      <widget class="QPushButton" name="workButton">
       <property name="text">
        <string>Run background work</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel work</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="prefsButton">
     <property name="text">
      <string>Preferences</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
